person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age?
+  end
```

Comment:
- [ ] The method `can_use_services?` should return true if the `of_age?` **OR** the `@parent_permission` variable is true, Right now it only checks for `of_age?`. Kindly refactor accordingly.

  **SUGGESTION:** You can refactor the function to be like so:
  ```
  def can_use_services?
    of_age? || @parent_permission
  end
  ```